WindowsServerDocs/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-client-vpn-connections.md

```diff
@@ -853,13 +846,25 @@ You should see the new VPN profile shortly.
 
 ## Configure the VPN client by using Intune
 
-To use Intune to deploy Windows 10 Remote Access Always On VPN profiles, you can configure the ProfileXML CSP node by using the VPN profile you created in the section [Create the ProfileXML configuration files](#create-the-profilexml-configuration-files), or you can use the base EAP XML sample provided below.
+To use Intune to deploy Windows 10 Remote Access Always On VPN profiles, you can configure the ProfileXML CSP node by using the VPN profile you created in the section [Create the ProfileXML configuration files](#create-the-profilexml-configuration-files) and modified using the rules below in "Output and Modify VPN_Profile.xml for Intune", or you can use the base EAP XML sample provided below.
```

Comment:
I don't see a "Output and ModifyVPN_Profile.xml for Intune", should this be Reformat VPN_Profile.xml for Intune?